src/FlightDisplay/FlightDisplayView.qml

```diff
@@ -728,6 +729,133 @@ Item {
         }
     }
 
+    //-- Annunciator panel
+    Rectangle {
+        visible:                    _showAnnunciatorPanel
+        id:                         annunciatorPanel
+        anchors.horizontalCenter:   parent.horizontalCenter
+        anchors.topMargin:          _toolsMargin
+        anchors.top:                parent.top
+        z:                          _mapAndVideo.z + 4
```

Comment:
The 4 seems arbitrary. Is there a way to get the count of all the items? Any concerns changes in the future could push the panel to be hidden?

Reply:
Good find, reduced this to be just above the map/video level.